pkg/client/options.go

```diff
@@ -521,6 +524,18 @@ func (c Continue) ApplyToList(opts *ListOptions) {
 	opts.Continue = string(c)
 }
 
+// UnsafeDisableCacheDeepCopy indicates not to deep copy objects during list objects in cache reader.
```

Comment:
```suggestion
// UnsafeDisableCacheDeepCopy indicates not to deep copy objects during list objects in cache reader.
// Be very careful with this, when enabled you must DeepCopy any object before mutating it, otherwise you
// will mutate the object in the cache.
```

Reply:
done